cmd/utils/flags.go

```diff
@@ -982,6 +989,9 @@ func SetP2PConfig(ctx *cli.Context, cfg *p2p.Config) {
 		cfg.NetRestrict = list
 	}
 
+	rpc.MaxRequestContentLength = ctx.GlobalInt(MaxRequestContentLengthFlag.Name)
+	grpc.MaxRequestContentLength = ctx.GlobalInt(MaxRequestContentLengthFlag.Name)
```

Comment:
If you won't use different values for `rpc` and `grpc`, how about declaring a shared variable at `common` package?

Reply:
Good point :)
I moved it to common package. I made a new file because there is no appropriate place to put it. Is this Okay?

Reply:
How about using `variables.go` instead of `rpc.go`? Since I'm not sure that we will have many rpc-related & shared variables in the future. How do you think?

Reply:
Your review is applied :)